spec/requests/customization_script_spec.rb

```diff
@@ -0,0 +1,37 @@
+RSpec.describe "Customization Scripts" do
+  describe "display a config pattern's details" do
+    context "without an appropriate role" do
+      it "forbids access to read config pattern" do
+        config_pattern = FactoryGirl.create(:customization_script)
+
+        api_basic_authorize
+        get api_customization_script_url(nil, config_pattern)
+
+        expect(response).to have_http_status(:forbidden)
+        expect(response.parsed_body["error"]).to include("kind" => "forbidden")
+      end
+    end
+
+    context "with valid id" do
+      it "returns both id and href" do
```

Comment:
just a NIT here, but it should return the whole config pattern resource, not just id and href, so this is a little misleading

Reply:
Should return all config pattern details(href, id, name, manager_id, created_at, description...)?

Reply:
@igorrran don't worry about testing all of the details...just update this to describe it as "returns the resource" or something similar. Thanks!!